packages/es-client/tests/test-granule-upsert.js

```diff
@@ -343,13 +555,50 @@ test('upsertGranule does not update "running" granule to "failed" for different
     esClient,
     updates,
     index: esIndex,
+  }, false);
+  const updatedEsRecord = await t.context.esGranulesClient.get(granule.granuleId);
+  t.like(updatedEsRecord, updates);
```

Comment:
I might be misunderstanding something but we want to compare the `updatedEsRecord` to be like `granule`, right? since it shouldn't update? I also think it might be worthwhile to check the status directly

Reply:
Hmm, perhaps the titles are misleading - this test is intended to allow the write, so it should be like the updates object.

Reply:
I understand now what the tests are doing and am not sure why I was confused 17 days ago, so sorry about that 😓